pkg/cli/cli.go

```diff
@@ -1578,6 +1590,9 @@ func newClient(config Config, flags *ServiceFlags, c *cobra.Command) (client.Cli
 	if flags.ServiceURL == unitTestServiceURL {
 		return &fakeClient{}, nil
 	}
+	if flags.ServiceURL == client.LocalDockerServiceURL {
+		return client.NewDockerClient(&client.ClientOptions{})
+	}
```

Comment:
Why is a different client implementation needed for local docker?

The different behavior should be implemented in the cloud orchestrator, with the client simply accessing its api in the same way other it does for other services.

Reply:
When used locally, rather than running the Cloud Orchestrator server locally and interact with it through the http api, we can interact with the Docker Engine directly from `cvdr` this makes it easier to manage cuttlefish instances when used locally. It's also easier for folks to reproduce the same environment they don't need to bother with running/maintaining a local Cloud Orchestrator service.

Reply:
no longer relevant.